static/js/signUp.js

```diff
@@ -42,7 +42,7 @@ function createUser(user) {
                 alert("This username already exists.");
             }
             else {
-                alert("Error occurred. User account was not created.");
+                alert("Error occurred. User account was not created. Site returned HTTP code: " + result.target.status);
```

Comment:
The error message that was being retuned to the user was not very helpful. "It didn't work". I added the HTTP code to the error message to reduce technical debt. One might wonder how this reduces technical debt. Well, once an application is deployed and users are using it, someone has to support the application. If the user reaches out to support and says "It didn't work, error was 'Error occurred. User account was not created'" then support has to go digging around and try to figure out exactly what caused the application to throw an error. Whereas if the HTTP response status code is included than the user can let support know exactly what the error is which will enable support to find and fix the bug faster which is an indicator of decreased technical debt. Read more about technical debt here: https://martinfowler.com/bliki/TechnicalDebt.html